frontend/src/metabase/visualizations/index.ts

```diff
@@ -60,7 +60,7 @@ export function registerVisualization(visualization: Visualization) {
       return;
     }
 
-    throw new Error(
+    console.warn(
```

Comment:
Visualization seems to be registering more than once. Looking into why this is the case.